src/clusterfuzz/_internal/bot/tasks/fuzz_task.py

```diff
@@ -1003,14 +1005,15 @@ def create_testcase(group, context):
   return testcase
 
 
-def filter_crashes(crashes):
+def filter_crashes(crashes: List[CrashInfo]) -> List[CrashInfo]:
   """Filter crashes based on is_valid()."""
   filtered = []
 
   for crash in crashes:
     if not crash.is_valid():
-      logs.log('Ignore crash (reason=%s, type=%s, state=%s).' %
-               (crash.get_error(), crash.crash_type, crash.crash_state))
+      logs.log((
+          f'Ignore crash (reason={crash.get_error()}, type={crash.crash_type}, '
+          f'state={crash.crash_state}, stacktrace={crash.crash_stacktrace}).'))
```

Comment:
Include the stacktrace as an additional kwarg. Similar to https://github.com/google/clusterfuzz/blob/4f8020c4c7ce73c1da0e68f04943af30bb5f0b32/src/clusterfuzz/_internal/system/process_handler.py#L259

Reply:
ah good call out. wasn't sure how this was handled. good to know thanks.

Reply:
fixed